homeassistant/components/rachio/.translations/en.json

```diff
@@ -0,0 +1,24 @@
+{
+  "config": {
+    "title": "Rachio",
+    "step": {
+      "user": {
+        "title": "Connect to your Rachio device",
+        "description" : "You will need the API Key from https://app.rach.io/. Select 'Account Settings, and then click on 'GET API KEY'.",
+        "data": {
+          "api_key": "The API key for the Rachio account.",
+          "hass_url_override": "If your instance is unaware of its actual web location (base_url).",
+          "manual_run_mins": "For how long, in minutes, to turn on a station when the switch is enabled."
```

Comment:
Let's move this to an option flow. Normal setup should focus on the things necessary to get it in Home Assistant. Nothing more.

Reply:
moved to option flow